os/arch/arm/src/amebasmart/amebasmart_reboot_reason.h

```diff
@@ -26,6 +26,7 @@ enum {
 	REBOOT_SYSTEM_SYS_RESET_CORE = REBOOT_BOARD_SPECIFIC2, /* System reset by Core */
 	REBOOT_SYSTEM_BOD_RESET      = REBOOT_BOARD_SPECIFIC3, /* Brownout reset */
 	REBOOT_SYSTEM_TZWD_RESET     = REBOOT_BOARD_SPECIFIC4, /* TrustZone Watch dog */
+	REBOOT_SYSTEM_CORE_FAULT     = REBOOT_BOARD_SPECIFIC5, /* Reset due to fault in KM4/KM0 */
```

Comment:
It looks CA32 core. Can we use another name instead of CORE? NETCORE? or something else?

Reply:
Changed to REBOOT_SYSTEM_NP_LP_FAULT to refer to fault coming from KM4/KM0